nautobot/core/api/views.py

```diff
@@ -460,8 +456,6 @@ def get(self, request):
                 "nautobot-version": settings.VERSION,
                 "plugins": plugins,
                 "python-version": platform.python_version(),
-                # 2.0 TODO: remove rq-workers-running
-                "rq-workers-running": RQWorker.count(get_rq_connection("default")),
```

Comment:
Should this API change be documented in `docs/installation/upgrading-from-nautobot-v1.md`